### ISSUE TITLE
[Quest][Borean Tundra] The Plains of Nasam

### ISSUE THREAD

**Issue by Smokey7777** [Stormforge-gg/Bugtracker-Wotlk-Beta#672]
**Description**: 
The vehicle is incredibly broken in multiple ways and the quest is unable to be completed for multiple reasons.
**Current behaviour**:
1. Scourge Leader cannot be identified, either in out of a vehicle. 
2. The vehicle's abilities are majorly broken. For instance, 1, the cannon, instead spawns a fireball that slowly floats in a southeast direction regardless of where you are aiming. Additionally, mana costs are so high in the vehicle (although the lack of mana regen is blizzlike) you can kill about 3 mobs with the battering ram before going oom, particularly the battering ram and buzzsaw should not cost vehicle mana. There's more, but it's so broken that it is beyond explanation, only comparing: https://www.youtube.com/watch?v=2_YqAxsZvYg
3. Injured Warsong Soldiers can be rescued with the 5th ability in the demolisher, but no quest credit is given. 
In other words, all 3 objectives of the quest cannot be done. (Although the second one, killing, can be done by getting out of the vehicle and manually killing 100 skeletons with auto attacks). 
**Expected behaviour**: 
The quest should be completable. 
**Revision**: 
1225

**Comment by Wolffenstein** [comment]
![image](https://user-images.githubusercontent.com/1576644/169859280-da93d302-cf34-43b7-a6fc-ca615730a6fa.png)
This part works for me, just gotta get close enough.

**Comment by Wolffenstein** [comment]
Also those spells DO cost mana in WotLK, they must have been changed in later patches.

**Comment by Wolffenstein** [comment]
Rest fixed in rev 1236.
